presto-docs/src/main/sphinx/optimizer/history-based-optimization.rst

```diff
@@ -0,0 +1,97 @@
+restrict_history_based_optimization_to_complex_query        Enable history based optimization only for complex queries, i.e. queries with join and aggregation      True
+history_input_table_statistics_matching_threshold           When the size difference between current table and history table exceed this threshold, do not match history statistics. When value is 0, use the default value set by hbo.history-matching-threshold       0
+=========================================================== =========================================================================================================================================================================================================== ===============
+
+Examples of HBO
+---------------
+
+An example of a query plan with HBO statistics is shown below. For a plan node, the estimation statistics will show source `HistoryBasedSourceInfo` when the statistics are from HBO.
+
+.. code-block:: text
+
+         Fragment 1 [HASH]                                                                                                                                            >
+             Output layout: [orderpriority, count]                                                                                                                    >
+             Output partitioning: SINGLE []                                                                                                                           >
+             Stage Execution Strategy: UNGROUPED_EXECUTION                                                                                                            >
+             - Project[PlanNodeId 392][projectLocality = LOCAL] => [orderpriority:varchar(15), count:bigint]                                                          >
+                     Estimates: {source: HistoryBasedSourceInfo, rows: 5 (117B), cpu: ?, memory: ?, network: ?}                                                       >
+                 - Aggregate(FINAL)[orderpriority][$hashvalue][PlanNodeId 4] => [orderpriority:varchar(15), $hashvalue:bigint, count:bigint]                          >
+                         Estimates: {source: HistoryBasedSourceInfo, rows: 5 (117B), cpu: ?, memory: ?, network: ?}                                                   >
+                         count := "presto.default.count"((count_8)) (1:50)                                                                                            >
+                     - LocalExchange[PlanNodeId 354][HASH][$hashvalue] (orderpriority) => [orderpriority:varchar(15), count_8:bigint, $hashvalue:bigint]              >
+                         - RemoteSource[2] => [orderpriority:varchar(15), count_8:bigint, $hashvalue_9:bigint]
+
+    
+Optimizations using HBO
+-----------------------
+
+DetermineJoinDistributionType & DetermineSemiJoinDistributionType
```

Comment:
```suggestion
DetermineJoinDistributionType and DetermineSemiJoinDistributionType
```
Avoid Latin abbreviations. For context, see the [recommended word list entry for &](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#) in the [Gitlab Documentation Style Guide](https://docs.gitlab.com/ee/development/documentation/styleguide/).